hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/processors/CustomMessageCallProcessor.java

```diff
@@ -187,7 +187,7 @@ private boolean isTokenCreation(MessageFrame frame) {
             return false;
         }
         var selector = frame.getInputData().slice(0, 4).toArray();
-        return CREATE_FUNCTIONS.stream().anyMatch(s -> Arrays.equals(s.selector(), selector));
+        return decoderMap.keySet().stream().anyMatch(s -> Arrays.equals(s.selector(), selector));
```

Comment:
`decoderMap` is a little hard to understand.  Can we come up with a different name?  Suggestion: `createFunctionSelectorsMap`

Reply:
Renamed